benchmarks/common/covfie/benchmark/lorentz.hpp

```diff
@@ -1,11 +1,7 @@
 /*
- * This file is part of covfie, a part of the ACTS project
+ * SPDX-FileCopyrightText: 2022 CERN
  *
- * Copyright (c) 2022 CERN
- *
- * This Source Code Form is subject to the terms of the Mozilla Public License,
- * v. 2.0. If a copy of the MPL was not distributed with this file, You can
- * obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
```

Comment:
Do you know why the `PackageName` does not appear here?

Reply:
Hmm probably because I overlooked that. Well spotted, I will fix it!

Reply:
In fact it looks like I only added the `PackageName` tag to `*.cpp` files twice instead of to `*.cpp` and `*.hpp` files.. sorry this is fixed now.

Reply:
Yes I define it as part of the linter. For example:
```
reuse annotate \
    --copyright="CERN" \
    --year 2022 \
    --license=MPL-2.0 \
    --style c \
    --merge-copyrights \
    -t acts-project \
    $(find . -name \*.cpp)
```